README.md

```diff
@@ -1,5 +1,16 @@
 # Elasticio CLI Tool
+## Table of Contents
+- [Installation](#Installation)
+- [Creating Test Fixtures](#Creating-Test-Fixtures)
+- [Commands](#Commands)
+  - [cmp:process](#cmp:process)
+  - [cmp:exec](#cmp:exec)
+  - [cmp:validate](#cmp:validate)
+- [Other Information](#Other-Information)
+
+
 ## Installation
+To install this tool, run:
```

Comment:
Can you add remarks about how to set up auto-completion? https://www.npmjs.com/package/caporal#shell-auto-completion

Related: it appears as though the `:` character breaks auto-completion.

Reply:
I am getting an error every time I try to set up autocompletion myself, that looks like the below.
Would it make more sense to leave the autocomplete to another issue for now since this is time-sensitive? I don't think this error is from our side, since I followed the caporal directions directly, but I am not sure?

```
events.js:187
      throw er; // Unhandled 'error' event
      ^

Error: EPIPE: broken pipe, write
    at Socket._write (internal/net.js:58:25)
    at doWrite (_stream_writable.js:431:12)
    at writeOrBuffer (_stream_writable.js:415:5)
    at Socket.Writable.write (_stream_writable.js:305:11)
    at CaporalTransport.log (C:\Users\Hannah Bulmer\elasticio-cli\node_modules\caporal\lib\logger.js:21:18)
    at transportLog (C:\Users\Hannah Bulmer\elasticio-cli\node_modules\winston\lib\winston\logger.js:234:15)
    at C:\Users\Hannah Bulmer\elasticio-cli\node_modules\async\lib\async.js:157:13
    at _each (C:\Users\Hannah Bulmer\elasticio-cli\node_modules\async\lib\async.js:57:9)
    at Object.async.each (C:\Users\Hannah Bulmer\elasticio-cli\node_modules\async\lib\async.js:156:9)
    at exports.Logger.Logger.log (C:\Users\Hannah Bulmer\elasticio-cli\node_modules\winston\lib\winston\logger.js:246:9)
    at exports.Logger.target.<computed> [as info] (C:\Users\Hannah Bulmer\elasticio-cli\node_modules\winston\lib\winston\common.js:54:18)
    at Program._handleCompletionCommand (C:\Users\Hannah Bulmer\elasticio-cli\node_modules\caporal\lib\program.js:262:21)
    at Program.parse (C:\Users\Hannah Bulmer\elasticio-cli\node_modules\caporal\lib\program.js:281:19)
    at Object.<anonymous> (C:\Users\Hannah Bulmer\elasticio-cli\elasticio:45:9)
    at Module._compile (internal/modules/cjs/loader.js:956:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
Emitted 'error' event on Socket instance at:
    at errorOrDestroy (internal/streams/destroy.js:108:12)
    at onwriteError (_stream_writable.js:446:5)
    at onwrite (_stream_writable.js:473:5)
    at Socket._write (internal/net.js:62:14)
    at doWrite (_stream_writable.js:431:12)
    [... lines matching original stack trace ...]
    at Program.parse (C:\Users\Hannah Bulmer\elasticio-cli\node_modules\caporal\lib\program.js:281:19) {
  errno: 'EPIPE',
  syscall: 'write',
  code: 'EPIPE'
}
```

Reply:
Sure.  We can write up another issue.  The only thing that we may want to change at the moment is to remove the `:` from the commands since that appears to create issues.